ProjectShop/Shipping/templates/all_shipping_addresses.html

```diff
@@ -0,0 +1,25 @@
+<ul>
+    <!-- Iterate over object_list -->
+    {% for object in object_list %}
+    <!-- Display Objects -->
+
+    <li><strong>Postal code: </strong> {{ object.postal_code }}</li>
+    <li><strong>Country (ISO code): </strong> {{ object.country }}</li>
+    <li><strong>Region: </strong> {{ object.region }}</li>
+    <li><strong>City: </strong> {{ object.city }}</li>
+    <li><strong>Post office №: </strong> {{ object.post_office }}</li>
+    <p>
+        <a href="/shipping/delete/{{object.id}}">Delete</a> -
+        <a href="/shipping/update/{{object.id}}">Update</a>
+    </p>
+    <hr>
+
+    <!-- If objet_list is empty  -->
+    {% empty %}
+    <li>No objects yet.</li>
+    {% endfor %}
+</ul>
+
+<form action="/shipping/add">
+    <input type="submit" value="Add another shipping address" />
+</form>
```

Comment:
Translation to HTML files should be added.
Please, visit the link for more information.
https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#internationalization-in-template-code
Should you have any questions, feel free to contact me.

Reply:
Okay, thank you for your comment, I'll add that